docs/core/turn-restrictions.md

```diff
@@ -0,0 +1,19 @@
+# How to work with Turn Restrictions
+
+GraphHopper supports [turn restrictions](http://wiki.openstreetmap.org/wiki/Relation:restriction).
+Currently, it is not possible to use turn restrictions with the speed mode (contraction hierarchies),
+but it's possible using the flexible mode (landmarks) and without any post-import optimization.
```

Comment:
can we call it flexible and hybrid mode and maybe linking to [the explanation](https://github.com/graphhopper/graphhopper#technical-overview)? Technically the landmarks algo is flexible but I avoid calling it flexible mode :)